modules/tutorials-for-data-scientists.adoc

```diff
@@ -68,8 +68,8 @@ ifndef::upstream[]
 | Creating a Jupyter notebook
 | Create a Jupyter notebook in JupyterLab.
 
-| Creating a Machine Learning Model using the NVIDIA GPU Add-on
```

Comment:
I initially took this out, but then found https://issues.redhat.com/browse/RHODS-12942 so I will make those updates separately/next.